release_notes.txt

```diff
@@ -0,0 +1,104 @@
+===============================================================================
+                              Swift Navigation, Inc.
+                      Piksi Multi Firmware Release Notes
+
+Product Page:      swiftnav.com/piksi-multi
+Firmware Download: support.swiftnav.com
+===============================================================================
+
+
+-------------------------------------------------------------------------------
+Release 1.1.26                                                       2017-05-16
+-------------------------------------------------------------------------------
+FEATURES:

```

Comment:
Should we enumerate new features at the top? Heading, ntrip, rtcm in, usb logging, etc.